phpstan.neon.dist

```diff
@@ -10,11 +10,14 @@ parameters:
         - tests
 
     bootstrapFiles:
-        - src/Core/DevOps/StaticAnalyze/phpstan-bootstrap.php
-    # run `php src/Core/DevOps/StaticAnalyze/phpstan-bootstrap.php` to create the container
+        - src/Core/DevOps/StaticAnalyze/phpstan-bootstrap.php # run `php src/Core/DevOps/StaticAnalyze/phpstan-bootstrap.php` to create the container
```

Comment:
The file is run as part of the bootstrap process of phpstan so it's created there anyway? Why would one need to manually run the command? Or am I misunderstanding ?